src/libraries/System.ComponentModel.Annotations/ref/System.ComponentModel.Annotations.cs

```diff
@@ -185,7 +189,9 @@ public sealed partial class KeyAttribute : System.Attribute
     [System.AttributeUsageAttribute(System.AttributeTargets.Field | System.AttributeTargets.Parameter | System.AttributeTargets.Property, AllowMultiple=false)]
     public partial class MaxLengthAttribute : System.ComponentModel.DataAnnotations.ValidationAttribute
     {
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Uses reflection to get the 'Count' property on types that don't implement ICollection. This 'Count' property may be trimmed.")]
         public MaxLengthAttribute() { }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Uses reflection to get the 'Count' property on types that don't implement ICollection. This 'Count' property may be trimmed.")]
```

Comment:
Earlier the message gave advice, e.g. "Ensure it is preserved."  Here we're just stating facts "This Count property may be trimmed".  It'd be nice to be consistent across all such messages.

Reply:
Thanks, fixed.